api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -73,9 +73,9 @@ type VirtualMachineImageStatus struct {
 	// Deprecated
 	PowerState    string `json:"powerState,omitempty"`
 
-	// GuestOSCustomizable indicates whether the VirtualMachineImage's osType is supported for
-	// customisation on the cluster
-	GuestOSCustomizable *bool  `json:"guestOsCustomizable,omitempty"`
+	// SupportedGuestOS indicates whether the VirtualMachineImage's osType is supported
+	// by the hosts in the cluster
```

Comment:
Supported by hosts *and VM Service*

Because if it's a windows vm for example, the hosts will support it, but we (vm service) won't.

Reply:
agreed. thanks.